nova/objects/instance.py

```diff
@@ -1717,6 +1745,22 @@ def get_counts(cls, context, project_id, user_id=None):
         return cls._get_counts_in_db_baremetalaware(context, project_id,
                                                     user_id=user_id)
 
+    @base.remotable_classmethod
+    def get_counts_by_project_and_az(cls, context, project_id=None):
+        """Get the counts of Instance objects in the database.
+
+        :param context: The request context for database access
+        :returns: A dict containing the project-scoped counts. For example:
+                    {'<project-id>': {
+                        '<availability-zone>: {
+                            'instances': <count across project in az>,
+                            'cores': <count across project in az>,
+                            'ram': <count across project in az>},
+                     ...}
```

Comment:
We don't return that structure, do we? It looks like we get a tuple as a key, so this would look more like this:
```
{('12341241241241', 'az-a'): {'instances': … }}
```

Reply:
I've fixed that, now the doc reflects the actual value returned (which is now a dict without tuples).